README.md

```diff
@@ -56,6 +56,7 @@ The following prerequisites are required for this application to function correc
     POSTGRES_USER=postgres<br>
     POSTGRES_PASSWORD=postgres<br>
     POSTGRES_DATABASE=gat_db<br>
+    DATABASE_AUTO_DEPLOY=1<br>
    ```
```

Comment:
What was the purpose of this line in the .env file?

Reply:
Craig recommended this after the issues we experienced first deploying my local database.
https://github.com/bcgov/gdx-agreements-tracker/blob/9a3468e0ff4b6cd132719aa37edc34bfc1902372/backend/src/database/databaseConnection.js#L66-L75